Scoop.js

```diff
@@ -1028,27 +1030,31 @@ export class Scoop {
     //
     // Generate summary page
     //
-    try {
-      const html = nunjucks.render('video-extracted-summary.njk', {
-        url: this.url,
-        now: new Date().toISOString(),
-        videoSaved,
-        metadataSaved,
-        subtitlesSaved,
-        availableVideosAndSubtitles,
-        metadataParsed
-      })
-
-      const url = 'file:///video-extracted-summary.html'
-      const httpHeaders = new Headers({ 'content-type': 'text/html' })
-      const body = Buffer.from(html)
-      const isEntryPoint = true
-      const description = `Extracted Video data from: ${this.url}`
-
-      this.addGeneratedExchange(url, httpHeaders, body, isEntryPoint, description)
-    } catch (err) {
-      this.log.warn('Error while creating exchange for file:///video-extracted-summary.html.')
-      this.log.trace(err)
+    if (videoSaved || metadataSaved || subtitlesSaved) {
```

Comment:
Nitpick: I could see this being a check before the try / catch block that tries to generate and record the summary. Both as a way to prevent nesting and getting more "precise" logs. 

Maybe like this?
```javascript 
if ((videoSaved || metadataSaved || subtitlesSaved) === false) {
  this.log.warn('Something something.')
  return
}
```

Where "something something" mentions that yt-dlp didn't return an exit code > 0, yet the capture was not successful?

Reply:
Sure thing.